Platform/macOS/Display/VMDisplayQemuMetalWindowController.swift

```diff
@@ -403,19 +403,24 @@ extension VMDisplayQemuMetalWindowController {
     func windowDidEnterFullScreen(_ notification: Notification) {
         isFullScreen = true
         if isFullScreenAutoCapture {
-            captureMouseToolbarButton.state = .on
```

Comment:
Why did you remove this line and 414?

Reply:
I think these lines are already present in captureMouse() and releaseMouse(). I just tried to check the behavior but I don't think the toolbar is ever visible in full screen when the mouse is captured.